### PR TITLE
finished implementing user functionality to create new lists

### PR DESCRIPTION
**Pull request by dmachineni**
- create api endpoints for logged in users to create new lists 
- created a button and form to allow users to create new lists from the frontend
- rendered user lists under the label "my lists" which is located above the generated list

- demo:
ezgif com-gif-maker (2)

### CODE REVIEW

File: ui/src/components/App/App.jsx [hunk 1]
```diff
@@ -49,9 +50,21 @@ export default function App() {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  const handleListDetails = (cat, subCat) => {
+  const handleListDetails = (cat, subCat, subCatRecipes) => {
+    console.log('handleListDetails function',cat,subCat)
```
Comment (DanielCKennedy) [inline ui/src/components/App/App.jsx]: remove console.logs

**Review by DanielCKennedy — CHANGES_REQUESTED** [review]
Looks good overall! Just remove your log statements

**Review by ksio2124fb — COMMENTED** [review]
node_modules should not be in source control

**Comment by ksio2124fb** [comment]
great progress!!

**Comment by DanielCKennedy** [comment]
> node_modules should not be in source control

Good call out, you may need to modify your .gitignore

**Review by DanielCKennedy — APPROVED** [review]

**Review by ksio2124fb — APPROVED** [review]

**Pull request merged**